### DISCUSSION TITLE
Cannot reapply for GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by pengzhi1998** [community/community#86086] [GitHub Education]
### Hi everyone,

Hi everyone, I'm facing a confusing situation here. My GitHub Education seems to be out-of-date now and I need to re-apply for a new [GitHub Student Developer Pack](https://education.github.com/pack). But when trying to do this, I was told `You were last verified as a student on . It is not necessary for you to re verify at this time.` What shall I do to deal with it? 

It seems like a deadlock to me: I need to join GitHub Global Campus, but when trying to join it (signing up for GitHub Student Developer Pack), I was told that I have already been registered and could not proceed.

Thank you in advance for any help!!

**Comment by DTS-11** [comment]
- I'd recommend you contact GitHub [support](https://support.github.com/contact).
- Also try checking your [GitHub Education](https://education.github.com) status. If there are any issues or if you need to update your information, there might be options available on this page.
- If there are any issues or if you need to update your information, there might be options available on this page.
- Sometimes, browser cache and cookies can cause issues. Try clearing your browser's cache and cookies and then attempt to sign up again.

**Reply by pengzhi1998** [reply]
Thank you so much for your quick reply! I have sent a request to GitHub Community. Hope I can get some help from them. Thanks a lot again!

**Reply by mishmanners** [reply]
If you don't get a response, you can also reach out to the [GitHub Education team on Twitter/X](https://twitter.com/GitHubEducation/). They are usually pretty responsive.
